src/pages/project/setdestination.js

```diff
@@ -8,10 +8,18 @@ module.exports = {
   componentDidMount: function () {
     // Handle button actions
     dispatcher.on('linkClicked', (event) => {
-      if (event.props.targetPageId && this.state.isPageZoomed) {
-        this.zoomToPage( this.pageIdToCoords(event.props.targetPageId) );
+      if (event.props.targetWebURL) {
+        if (window.Platform) {
+          window.Platform.openExternalUrl(event.props.targetWebURL);
+        } else {
+          document.location.href = event.props.targetWebURL;
+        }
       } else {
-        this.highlightPage(event.props.targetPageId, 'selected');
+        if (event.props.targetPageId && this.state.isPageZoomed) {
+          this.zoomToPage( this.pageIdToCoords(event.props.targetPageId) );
+        } else {
+          this.highlightPage(event.props.targetPageId, 'selected');
```

Comment:
Oh, I see the results of highlightPage, this is for setting destination and "highlighting" the current page.